includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -808,6 +808,8 @@ public function get_proxy_permissions_url() {
 			$query_args['site_id'] = $credentials->web->client_id;
 		}
 
+		$query_args['application_name'] = rawurlencode( 'wordpress/google-site-kit/' . GOOGLESITEKIT_VERSION );
```

Comment:
Now that we use this in three places, all in this class, can you add a private `get_application_name()` method that returns this? It's more error-proof having that in one place to ensure it always remains consistent.

Reply:
👍

Reply:
added in https://github.com/google/site-kit-wp/pull/1572/commits/a57f22a92219aefa13e8722f1e4b3ab522bf80e8